pkg/gadgets/trace/dns/types/dns.go

```diff
@@ -22,6 +22,7 @@ import (
 type Event struct {
 	eventtypes.Event
 
+	Qr         string `json:"qr,omitempty" column:"qr,width:2,fixed"`
```

Comment:
We should use an enum for this so that users that inspect this file, easily understand that `Qr` can be either `R` or `Q`.

Reply:
Done!